### ISSUE TITLE
libffmpeg.dylib image not found on Mac OSX

### ISSUE THREAD

**Issue by tedneward** [atom/electron#4765]
* Electron version: 0.37.0
* Operating system: Mac OSX latest

Repro: Node v5.8.0. Follow QuickStart. Using Electron 0.37.0, "electron index.js" yields:

````
Teds-MBP15:helloworld ted$ npm start

> hello-world@0.1.0 start /Users/ted/Projects/Presentations.hg/Content/Electron/code/helloworld
> electron index.js

dyld: Library not loaded: /usr/local/lib/libffmpeg.dylib
  Referenced from: /Users/ted/Projects/Presentations.hg/Content/Electron/code/helloworld/node_modules/electron-prebuilt/dist/Electron.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Electron Framework
  Reason: image not found
````

Downgrading to 0.36.* works flawlessly.

**Comment by z-ax** [comment]
+1

**Comment by patrickpereira** [comment]
+1

**Comment by frikadel** [comment]
+1

**Comment by leftstick** [comment]
+1

**Comment by gsantome** [comment]
+1

**Comment by ZakariaRidouh** [comment]
@z-ax @patrickdet @frikadel @leftstick @gsantome Please use the new Github reactions feature instead of spamming the OP with useless +1's. Thank you.

**Comment by tomvillp** [comment]
as a temporary fix (to allow running v0.37.0), copy 

> Electron.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Libraries/libffmpeg.dylib

and paste it to 

> /usr/local/lib
